idaes/models/flowsheets/methanol_flowsheet.py

```diff
@@ -0,0 +1,736 @@
+        if hasattr(unit, 'pressure_balance'):
+            for t, c in unit.pressure_balance.items():
+                iscale.constraint_scaling_transform(c, 1e-5, overwrite=False)
+        if hasattr(unit, 'sum_of_phase_fractions'):
+            for t, c in unit.sum_of_phase_fractions.items():
+                iscale.constraint_scaling_transform(c, 1e2, overwrite=False)
+        if hasattr(unit, "material_accumulation_disc_eq"):
+            for (t, p, j), c in unit.material_accumulation_disc_eq.items():
+                iscale.constraint_scaling_transform(c, 1, overwrite=False)
+
+        if hasattr(unit, "energy_accumulation_disc_eq"):
+            for (t, p), c in unit.energy_accumulation_disc_eq.items():
+                iscale.constraint_scaling_transform(c, 1e-3, overwrite=False)
+
+        if hasattr(unit, "element_accumulation_disc_eq"):
+            for (t, e), c in unit.element_accumulation_disc_eq.items():
+                iscale.constraint_scaling_transform(c, 1, overwrite=False)
+
+    # equality constraints between ports at Arc sources and destinations
+    for arc in m.fs.component_data_objects(Arc, descend_into=True):
+        for c in arc.component_data_objects(Constraint, descend_into=True):
+            if hasattr(unit, "enth_mol_equality"):
+                for t, c in unit.enth_mol_equality.items():
+                    iscale.constraint_scaling_transform(c, 1e-3,
+                                                        overwrite=False)
+            if hasattr(unit, "flow_mol_equality"):
+                for t, c in unit.flow_mol_equality.items():
+                    iscale.constraint_scaling_transform(c, 1,
+                                                        overwrite=False)
+            if hasattr(unit, "mole_frac_comp_equality"):
+                for (t, j), c in unit.mole_frac_comp_equality.items():
+                    iscale.constraint_scaling_transform(c, 1e2,
+                                                        overwrite=False)
+            if hasattr(unit, "pressure_equality"):
+                for t, c in unit.pressure_equality.items():
+                    iscale.constraint_scaling_transform(c, 1e-5,
+                                                        overwrite=False)
+
+    iscale.calculate_scaling_factors(m)
+
+
+def initialize_flowsheet(m):
+
+    # Initialize and solve flowsheet
+
+    print('')
+    m.fs.M101.initialize()
+    propagate_state(arc=m.fs.s02)  # mixer to compressor
+
+    m.fs.C101.initialize()
+    propagate_state(arc=m.fs.s03)  # compressor to heater
+
+    m.fs.H101.initialize()
+    propagate_state(arc=m.fs.s04)  # heater to reactor
+
+    m.fs.R101.initialize()
+    propagate_state(arc=m.fs.s05)  # reactor to turbine
+
+    m.fs.T101.initialize()
+    propagate_state(arc=m.fs.s06)  # turbine to cooler
+
+    m.fs.H102.initialize()
+    propagate_state(arc=m.fs.s07)  # cooler to flash
+
+    m.fs.F101.initialize()
+
+
+def add_costing(m):
```

Comment:
As I noted in the previous review, this is using the old style costing methods. If this is going into `idaes.models`, then I will need to insist that this is updated to the new style before it can be merged.

I would strongly encourage you to make the change anyway, as otherwise you will just need to update this in the next few months when we fully deprecate the old-style approach.